src/robotoff.js

```diff
@@ -126,5 +126,33 @@ export default {
     return axios.get(
       `${ROBOTOFF_API_URL}/images/logos?logo_ids=${logoIds.join(",")}`
     )
+  },
+
+  getNutritionValueFromImage(productCode, language, imageOcrUrl) {
+
+    var ocrUrlSubString = imageOcrUrl.split("/");
+
+    let keyName = `nutrition_${language}`;
+    
+    return axios.get(
+      `${OFF_API_URL}/product/${productCode}.json?fields=product_name,images`
+
+    ).then(result => {
```

Comment:
This data fetching does not seem to be useful, because for every product in the buffet you already have the following properties `code,lang,image_nutrition_url,product_name,images` (see in the method `getProducts`)

You can reuse it by passing the `images` or `imgid` as an argument instead of `getProducts` which is not used anymore